libvips/arithmetic/measure.c

```diff
@@ -253,9 +253,9 @@ vips_measure_init( VipsMeasure *measure )
 }
 
 /**
- * vips_measure:
+ * vips_measure: (method)
  * @in: image to measure
- * @out: array of measurements
+ * @out: (out)(array): array of measurements
```

Comment:
Is (array) correct? This is an output image whose pixels are the array elements, if you see what I mean.